sickbeard/providers/nyaatorrents.py

```diff
@@ -61,10 +61,10 @@ def findSearchResults(self, show, episodes, search_mode, manualSearch=False, dow
         return generic.TorrentProvider.findSearchResults(self, show, episodes, search_mode, manualSearch, downCurQuality)
 
     def _get_season_search_strings(self, ep_obj):
-        return show_name_helpers.makeSceneShowSearchStrings(self.show, anime=True)
+        return [x for x in show_name_helpers.makeSceneSeasonSearchString(self.show, ep_obj)]
 
     def _get_episode_search_strings(self, ep_obj, add_string=''):
-        return self._get_season_search_strings(ep_obj)
+        return [x for x in show_name_helpers.makeSceneSearchString(self.show, ep_obj)]
```

Comment:
[x + ' ' +  add_string if add_string else x for x in show_name_helpers.makeSceneSearchString(self.show, ep_obj)]